### DISCUSSION TITLE
External converter for TV02-Zigbee radiator valve

### DISCUSSION THREAD

**Discussion by dkovacic74** [Koenkk/zigbee2mqtt#13729] [Devices]
Does anyone have external converter for this valve https://www.zigbee2mqtt.io/devices/TV02-Zigbee.html ?
Zigbee model: "TS0601", manufacturer name: "_TZE200_hue3yfsn".

I use zigbee2mqtt in Home assistant, and it said it is unsupported.

Regards, 
DK

**Comment by luetter82** [comment]
Hey @dkovacic74 ,

do you find a solution for your problem? 
i bought the same devices and now i see the log in zigbee2mqtt:
Received message from unsupported device with Zigbee model 'TS0601' and manufacturer name '_TZE200_hue3yfsn'

best regards

Mathias
